withdraw-by-check/REPWRITERSPECS/BANNO.CHECK.WITHDRAW.V1.POW

```diff
@@ -469,7 +541,18 @@ PROCEDURE FINDACCOUNT
        TMPCHR=LOAN:DESCRIPTION
        CALL CLEANUPTEXT
        SLDESCRIPTION=TMPCHR
-       SLAVAILABLEBAL=LOAN:AVAILABLECASHADVANCE
+
+       IF (LOAN:CREDLIMITEXPIRATION=DATENULL OR
+           LOAN:CREDLIMITEXPIRATION>SYSTEMDATE) THEN
+        DO
+         IF LOAN:LOANCODE=3 AND
+         (LOAN:INTERESTTYPE>=10 AND
+          LOAN:INTERESTTYPE<=18999) THEN
```

Comment:
Credit Card Interest types appear to be 10 - 8999, then 9000-9999 is reserved for future use. Any reason for 18999?

Reply:
Need to reapply correction